app/frontend/src/features/userPage/UserHeader.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { User } from "../../pb/api_pb";
+import { Box, makeStyles, Typography } from "@material-ui/core";
+import PageTitle from "../../components/PageTitle";
+import ScoreBar from "../../components/ScoreBar";
+import { CouchIcon, EditIcon } from "../../components/Icons";
+import Avatar from "../../components/Avatar";
+import { useTypedSelector } from "../../store";
+import { Link } from "react-router-dom";
+import { profileRoute } from "../../AppRoutes";
+import Button from "../../components/Button";
+import { timeAgo } from "../../utils/timeAgo";
+import { hostingStatusLabels } from "../../constants";
+import TextBody from "../../components/TextBody";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginBottom: theme.spacing(2),
+  },
+  avatar: {
+    maxWidth: 200,
+    marginLeft: "auto",
+    marginRight: "auto",
+  },
+  name: {
+    marginBottom: theme.spacing(1),
+  },
+  hostStatusIcon: {
+    //workaround for https://github.com/cssinjs/jss/issues/1414
+    //still present despite upgrading to 10.5.0
+    marginInlineEnd: `${theme.spacing(1)}px`,
```

Comment:
You might not need the workaround if the `resolutions` fix in `package.json` I suggest on the file works.

Reply:
Still happens for me 🤷